packages/cli/src/databases/entities/ExecutionAnnotation.ts

```diff
@@ -18,18 +19,25 @@ export class ExecutionAnnotation {
 	@PrimaryGeneratedColumn()
 	id: number;
 
+	/**
+	 * This field stores the up- or down-vote of the execution by user.
+	 */
 	@Column({ type: 'varchar', nullable: true })
 	vote: AnnotationVote | null;
 
+	/**
+	 * Custom text note added to the execution by user.
+	 */
 	@Column({ type: 'varchar', nullable: true })
-	note: string;
+	note: string | null;
 
 	@RelationId((annotation: ExecutionAnnotation) => annotation.execution)
 	executionId: string;
```

Comment:
Could we also include a unique constraint for this, to enforce the data model integrity on DB level (i.e. that there is at most one annotation per execution)

Reply:
There is a unique index on executionId column for annotations, shouldn't this be enough? Anyway, I've added Index decorator on `execution` in typeorm entity as well 👌